tests/pkg_tensorflow-lite/main.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ *
+ * @file
+ * @brief       TensorFlow Lite test application
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+void setup();
```

Comment:
Can you comment on why you need `setup` and `loop`

Reply:
Done